staffeln/conductor/manager.py

```diff
@@ -23,6 +24,7 @@ def __init__(self, worker_id, conf):
         self.conf = conf
         self.ctx = context.make_context()
         self.controller = backup.Backup()
+        self.inc_count = 0

```

Comment:
I think it will make more sense if we start with =full_backup_depth
because it depends on exist backup to increment. So first backup should go with full backup.

Also We can do the following in this patch or in follow on, which I'm happy to work on too:
I think we should go with a separate count for each backup instead of a central count. For example, if a new volume is detected on the incremental turn, it will try to do incremental backup for its very first backup. Also this is not scalable.
To consider scalable, we should count in each volume backup record instead of here.